.eslintrcts.json

```diff
@@ -13,72 +13,73 @@
-    "max-len": [1, 120],
-    "brace-style": [1, "stroustrup", { "allowSingleLine": true }],
-    "comma-spacing": [1, {"before": false, "after": true}],
-    "comma-style": [1, "last"],
-    "computed-property-spacing": [1, "never"],
-    "consistent-this": [1, "self"],
-    "eol-last": 1,
-    "new-cap": 0,
-    "new-parens": 1,
-    "no-mixed-spaces-and-tabs": 1,
-    "no-nested-ternary": 1,
-    "no-spaced-func": 1,
-    "no-trailing-spaces": 1,
-    "keyword-spacing": [1, {"before": true, "after": true}],
-    "space-before-blocks": [1, "always"],
-    "@typescript-eslint/explicit-function-return-type": 0,
+    "consistent-return": "error",
+    "curly": "error",
+    "default-case": "error",
+    "guard-for-in": "error",
+    "no-alert": "error",
+    "no-caller": "error",
+    "no-cond-assign": "error",
+    "no-constant-condition": "error",
+    "no-debugger": "error",
+    "no-dupe-args": "error",
+    "no-dupe-keys": "error",
+    "no-duplicate-case": "error",
+    "no-else-return": "error",
+    "no-empty": "error",
+    "no-empty-character-class": "error",
+    "no-eq-null": "error",
+    "no-ex-assign": "error",
+    "no-extend-native": "error",
+    "no-extra-boolean-cast": "error",
+    "no-extra-semi": "error",
+    "no-fallthrough": "error",
+    "no-func-assign": "error",
+    "no-invalid-regexp": "error",
+    "no-invalid-this": "error",
+    "no-irregular-whitespace": "error",
+    "no-lone-blocks": "error",
+    "no-loop-func": "error",
+    "no-multi-spaces": "error",
+    "no-new-wrappers": "error",
+    "no-new": "error",
+    "no-octal": "error",
+    "no-negated-in-lhs": "error",
+    "no-obj-calls": "error",
+    "no-redeclare": "error",
+    "no-regex-spaces": "error",
+    "no-return-assign": "error",
+    "no-self-compare": "error",
+    "no-shadow": "error",
+    "no-unreachable": "error",
+    "no-unexpected-multiline": "error",
+    "no-unused-expressions": "error",
+    "no-use-before-define": "off",
+    "use-isnan": "error",
+    "valid-typeof": "error",
+    "array-bracket-spacing": ["error", "never"],
+    "max-len": ["error", 120],
+    "brace-style": ["error", "stroustrup", { "allowSingleLine": true }],
+    "comma-spacing": ["error", {"before": false, "after": true}],
+    "comma-style": ["error", "last"],
+    "computed-property-spacing": ["error", "never"],
+    "consistent-this": ["error", "self"],
+    "eol-last": "error",
+    "new-cap": "error",
+    "new-parens": "error",
+    "no-mixed-spaces-and-tabs": "error",
+    "no-nested-ternary": "error",
+    "no-spaced-func": "error",
+    "no-trailing-spaces": "error",
+    "keyword-spacing": ["error", {"before": true, "after": true}],
+    "space-before-blocks": ["error", "always"],
+    "@typescript-eslint/explicit-function-return-type": "off",
     "camelcase": ["error", { "properties": "never" }],
-    "@typescript-eslint/ban-ts-ignore": 0,
-    "no-unused-vars": 0,
+    "@typescript-eslint/ban-ts-ignore": "off",
+    "no-unused-vars": "off",
```

Comment:
🤨

Reply:
?

Reply:
I am wondering why unused variables need to be allowed for now, but I don't think it's a blocker.

Reply:
Oh, it was before. I just changed it to a more readable string.